### PR TITLE
Started Building Test Environment

### PR DESCRIPTION
**Pull request by elliesch**

### CODE REVIEW

**Comment by dcuffari** [comment]
Looks good.

**Comment by elliesch** [comment]
@dcuffari I've added more commits to show the whole code. Thanks for looking these over!

**Comment by elliesch** [comment]
@dcuffari I've added another commit to run a trivial unit test

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kelle) [inline .idea/vcs.xml]: I dont think this file should be in the repo.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: python
+
+python:
+- 3.3.0
```
Comment (kelle) [inline .travis.yml]: Version 3.3 is failing in Travis. Not sure why but should be fixed.

File: test_typing_kit.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from unittest import TestCase
```
Comment (kelle) [inline test_typing_kit.py]: Is this necessary? I don't know what this package is.

File: test_typing_kit.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+
+class TestTyping_kit(TestCase):
+    def test_typing_kit(self):
```
Comment (kelle) [inline test_typing_kit.py]: I look forward to seeing more tests in the  future!

File: .travis.yml (hunk 2 above)
Comment (dr-rodriguez) [inline .travis.yml]: I remember astrodbkit having similar issues with Python 3.3, I think it was some incompatibility with numpy. I wouldn't test against 3.3 anyways, maybe just 2.7, 3.5, and 3.6.

File: test_typing_kit.py (hunk 3 above)
Comment (elliesch) [inline test_typing_kit.py]: I'm not sure! I'm going to look through Adrian's testing and decide.

File: .travis.yml (hunk 2 above)
Comment (elliesch) [inline .travis.yml]: Thanks @dr-rodriguez!

**Pull request merged**